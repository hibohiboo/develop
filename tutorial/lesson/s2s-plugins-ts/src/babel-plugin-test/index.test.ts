import plugin from '.';
import * as pluginTester from 'babel-plugin-tester';

pluginTester({
  plugin,
  tests: [
    {
      title: 'snapshot test',
      code: `var a = 1 + 1`,
      snapshot: true
    },
    {
      title: 'no change',
      code: `var a = 1 * 1;`,
      output: `var a = 1 * 1;`,
    },
  ],
});