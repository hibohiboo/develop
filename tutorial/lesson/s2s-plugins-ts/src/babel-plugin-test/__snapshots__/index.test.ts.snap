// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test snapshot test: snapshot test 1`] = `
"
var a = 1 + 1

      ↓ ↓ ↓ ↓ ↓ ↓

var a = 1 * 1;
"
`;
